charts/hono/templates/_helpers.tpl

```diff
@@ -206,7 +206,11 @@ command:
   {{- required ".Values.adapters.commandAndControlSpec MUST be set if example AMQP Messaging Network is disabled" .dot.Values.adapters.commandAndControlSpec | toYaml | nindent 2 }}
 {{- end }}
 tenant:
-{{- if .dot.Values.deviceRegistryExample.enabled }}
+{{- if .dot.Values.adapters.tenantSpec }}
+  {{- range $key, $value := .dot.Values.adapters.tenantSpec }}
+  {{ $key }}: {{ $value }}
+  {{- end }}
```

Comment:
how about simply using
```
{{- .dot.Values.adapters.tenantSpec | toYaml | nindent 2 }}
```

Reply:
That's cleaner indeed. Also replaced other occurances.